DEVNOTE.md

```diff
@@ -45,3 +47,37 @@ To setup it:
 4. In *Credentials* section, set up an `ssh` connection to the UTBot docker container you have. Consider enabling connection via OpehSSH authentification agent to ease files synchronization.
 5. All other fields should fill in automatically.
 
+
+## Troubleshooting the build
+### read -i invalid option
+You can experience this on Mac OS:
+```shell
+read: -i: invalid option
+```
+It is because Mac OS has too old bash version. Upgrade bash version and set it to default:
```

Comment:
https://stackoverflow.com/questions/44662044/read-i-invalid-option-in-shell-on-osx
https://itnext.io/upgrading-bash-on-macos-7138bd1066ba